### DISCUSSION TITLE
AXEvent from Template

### DISCUSSION THREAD

**Discussion by Cacsjep** [AxisCommunications/acap-native-sdk-examples#211] [Q&A] [answered]
Hey

I am wondering how ax_event_handler_declare_from_template() works. The Docs says that second param is:

> event_template	The name of the event template

But how should i package the Template eg. where shoud I put the Template XML? Just in package root? I cant found documention about how to handle the Template File.

Thanks

**Comment by pataxis** [comment, marked as answer]
Hi @Cacsjep , sorry, this one got lost for quite a while. Templates are poorly documented and are not recommended to use. I'm not sure they will even work from AXIS OS 12.0 when root privileges are removed. Hopefully this can be better documented in the future.
